src/Model/User.php

```diff
@@ -466,19 +466,30 @@ public static function create(array $data)
 		// collapse multiple spaces in name
 		$username = preg_replace('/ +/', ' ', $username);
 
-		if (mb_strlen($username) > 48) {
-			throw new Exception(L10n::t('Please use a shorter name.'));
+		$username_min_length = max(1, min(255, intval(Config::get('system', 'username_min_length', 0))));
+		$username_max_length = max(1, min(255, intval(Config::get('system', 'username_max_length', 0))));
```

Comment:
It will most likely never get relevant, but the fallback default value for the `username_max_length` should be larger then `0` I think. For the `username_min_length` I would go to `1` (which it is via the `max` check) and for the max length maybe the old value of 48.

Reply:
Fair enough.